src/modules/moonraker/filesystem/home/pi/klipper_config/moonraker.conf

```diff
@@ -6,6 +6,7 @@ enable_debug_logging: False
 [file_manager]
 log_path: ~/klipper_logs
 config_path: ~/klipper_config
+enable_object_processing: True
```

Comment:
We might want to add a note here like MainsailOS has:

https://github.com/mainsail-crew/MainsailOS/blob/develop/src/modules/moonraker/filesystem/home/pi/klipper_config/moonraker.conf#L12